apps/web/src/lib/wagmi/hooks/rewards/hooks/useAcceptAngleConditions.ts

```diff
@@ -111,9 +112,15 @@ export const useAcceptAngleConditions = (
 
   const onError = useCallback((e: Error) => {
     if (e instanceof Error) {
```

Comment:
Callback says e: Error but you check again, unsure what is possible here. Looks like some cleanup is needed.